tests/test_encryption.py

```diff
@@ -317,6 +318,32 @@ def test_decrypt_token_v2_custom_now(self):
         self.assertEqual(_example_id, result.uid2)
 
 
+    def test_encrypt_token_v3(self):
+        uid2 = "Y29keWlzZ3JlYXQ="
```

Comment:
this uid here and in test_encrypt_token_v4 function below doesn't look like legit id? can we use _example_id instead?

and can we rename test_encrypt_token_v3 and test_encrypt_token_v4 to smoke_test_token_v3 and smoke_test_token_v4 to be more consistent with other sdk's naming?

Reply:
test have to start with test_ but I named them test_smoke